.github/workflows/Bulldozer

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
```

Comment:
Where did you create your classes? Let’s  move it to the correct package  `src/main/java/core/basesyntax`